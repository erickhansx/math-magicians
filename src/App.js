import React from 'react';
import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
import Calculator from './components/Calculator';
import NavBar from './components/NavBar';

class App extends React.Component {
  constructor(props) {
    super(props);
    this.state = {};
  }

  render() {
    return (
      <Router>
        <NavBar />

        <div className="App">
          <div className="main-holder">
            <Routes>
              <Route exact path="/Calculator" element={<Calculator />} />
            </Routes>
          </div>
        </div>
      </Router>
    );
  }
}

export default App;
