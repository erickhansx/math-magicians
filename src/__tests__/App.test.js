import renderer from 'react-test-renderer';
import App from '../App';

test('Quote Renders', () => {
  const AppRender = renderer.create(<App />).toJSON();
  expect(AppRender).toMatchSnapshot();
});
