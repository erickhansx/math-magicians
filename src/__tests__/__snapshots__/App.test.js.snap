// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote Renders 1`] = `
<div
  className="App"
  style={
    Object {
      "color": "grey",
      "fontSize": "20px",
      "margin": "20px",
    }
  }
>
  <div
    className="navbar"
  >
    <div
      className="navbar__headers"
    >
      <h1>
        Math Magicians
      </h1>
    </div>
    <div
      className="navbar__menu"
    >
      <a
        className="sc-bcXHqe krzFeG"
        href="/Home"
        onClick={[Function]}
      >
        Home
      </a>
      <a
        className="sc-bcXHqe krzFeG"
        href="/Calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
      <a
        className="sc-bcXHqe krzFeG"
        href="/Quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </div>
  </div>
  <div
    className="main-holder"
  >
    <div
      className="home"
    >
      <h2>
        Welcome to our page!
      </h2>
      <p>
        Lorem ipsum dolor sit amet consectetur, adipisicing elit. Dicta doloribus mollitia aut delectus. Harum illum qui, voluptatibus amet, ipsum adipisci ipsam maiores a, fugit praesentium reprehenderit soluta. Incidunt, soluta quo. Consectetur nulla animi cum numquam blanditiis. Magnam amet quidem soluta explicabo numquam expedita exercitationem laboriosam eum quas hic earum pariatur consectetur optio distinctio, quaerat eos necessitatibus, voluptatibus praesentium saepe reiciendis. Ipsum nostrum distinctio consequatur. Dolores id sequi nobis dignissimos, consectetur velit nihil architecto perferendis exercitationem excepturi eos quisquam, deserunt temporibus, et autem fugit rem placeat aut? Facilis voluptatum aliquid rerum. Porro ab sit velit quam consequatur eveniet veritatis, ducimus dolor! Illo, nemo. Itaque ut dicta porro quas aspernatur! Ad necessitatibus sit qui placeat obcaecati aliquam voluptatem accusantium quis harum sint! Quasi officiis repellat in numquam officia. Maxime, at. Deserunt quisquam quasi praesentium nobis, aut officia quas commodi aperiam dolores ab suscipit qui, fuga molestias rem animi, dicta est earum laudantium! Amet officiis sunt distinctio? Recusandae, incidunt. Maiores impedit praesentium quibusdam, quod unde dolorem quae atque, ab animi vero asperiores placeat natus, itaque officiis dignissimos corporis est iste cupiditate veniam molestias? Error itaque explicabo fuga, sapiente saepe magnam dicta doloremque, alias quasi inventore labore, quas omnis maiores. Lorem ipsum dolor sit amet consectetur, adipisicing elit. Dicta doloribus mollitia aut delectus. Harum illum qui, voluptatibus amet, ipsum adipisci ipsam maiores a, fugit praesentium reprehenderit soluta. Incidunt, soluta quo. Consectetur nulla animi cum numquam blanditiis. Magnam amet quidem soluta explicabo numquam expedita exercitationem laboriosam eum quas hic earum pariatur consectetur optio distinctio, quaerat eos necessitatibus, voluptatibus praesentium saepe reiciendis. Ipsum nostrum distinctio consequatur. Dolores id sequi nobis dignissimos, consectetur velit nihil architecto perferendis exercitationem excepturi eos quisquam, deserunt temporibus, et autem fugit rem placeat aut? Facilis voluptatum aliquid rerum. Porro ab sit velit quam consequatur eveniet veritatis, ducimus dolor! Illo, nemo. Itaque ut dicta porro quas aspernatur! Ad necessitatibus sit qui placeat obcaecati aliquam voluptatem accusantium quis harum sint! Quasi officiis repellat in numquam officia. Maxime, at. Deserunt quisquam quasi praesentium nobis, aut officia quas commodi aperiam dolores ab suscipit qui, fuga molestias rem animi, dicta est earum laudantium! Amet officiis sunt distinctio? Recusandae, incidunt. Maiores impedit praesentium quibusdam, quod unde dolorem quae atque, ab animi vero asperiores placeat natus, itaque officiis dignissimos corporis est iste cupiditate veniam molestias? Error itaque explicabo fuga, sapiente saepe magnam dicta doloremque, alias quasi inventore labore, quas omnis maiores.
      </p>
    </div>
  </div>
</div>
`;
